From 67b53a8a59946b1691b11bb3b632a31692b43375 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 24 Sep 2014 17:42:24 -0400
Subject: [redhat] configs: Enable Berkeley Packet Filter filtering for power LE

Message-id: <20140924174224.5118.14776.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 96705
O-Subject: [PATCH RHEL7.1 BZ1145615 1/2] redhat Kconfig: Enable Berkeley Packet Filter filtering for power LE
Bugzilla: 1145615
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1145615

Description:
------------
Enable Berkeley Packet Filter filtering for power LE.

RHEL Version Found:
-------------------
7.1

Upstream Status:
----------------
n/a

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/powerpc64le/CONFIG_BPF_JIT b/redhat/configs/generic/powerpc64le/CONFIG_BPF_JIT
index 7905dce..5f9bba7 100644
--- a/redhat/configs/generic/powerpc64le/CONFIG_BPF_JIT
+++ b/redhat/configs/generic/powerpc64le/CONFIG_BPF_JIT
@@ -1 +1 @@
-# CONFIG_BPF_JIT is not set
+CONFIG_BPF_JIT=y
-- 
1.7.1